### DISCUSSION TITLE
I can't join Student Developer Pack

### DISCUSSION THREAD

**Discussion by conghuan0502** [community/community#114393] [GitHub Education]
### Hi everyone,

Well, currently, I'm learning at Greenwich University, Danang Campus (Vietnam). I tried to join the student pack, but I had a problem taking a photo with my student ID card. In the beginning, I took a photo using my laptop webcam and it didn't give high resolution. Then I tried to use my phone to take a photo. Unfortunately, I can only use my front camera and can't use my back camera (for some reason). So I can't use my Student ID card anymore. Anyone can help me to join this student pack? I'm kinda rush right now. Thanks a lot!

**Comment by devashish2024** [comment]
- You can apply with your school's email address
- You can take any ID proof instead too (like a leave application? Or a letter for applying for Github Student Dev pack, its better)
- Try another phone (your family's anyone, they might just help you!)

**Reply by conghuan0502** [reply]
Well, so I can create a new account using my school email right? And what if I wanna take a picture of my  University moodle? Is it work?

**Reply by devashish2024** [reply]
University model is not a valid proof. You do not need to make a account with school's email (but it is great if your school provides emails like yourname@school.com) but it will be better to have.

**Reply by devashish2024** [reply]
If you wish, go [here](https://education.github.com/discount_requests/application) and add a new email address issued by your school to have chances of faster and more likely to be approved review.
![image](https://github.com/community/community/assets/83082760/c25859b7-9216-4d05-949e-cf72999d6c4d)

**Reply by conghuan0502** [reply]
I mean, I already added my school email and used it. Tbh, I got 2 school emails, one provided by the Vietnam campus, and one provided by the UK campus. So what I can do now, and how long does it take to process my application?

**Reply by devashish2024** [reply]
It takes 2 to 4 days, however sometimes its speedy and sometimes too slow. After approved, it can take from 1 day to 4 and more sometimes to give all the perks.

In the meanwhile, I got approved after applying in 3 days, and 3 days later got verified/got all perks.

> it appears like it might be the answer, so feel free to mark this as answer if I solved your problem

**Reply by conghuan0502** [reply]
So if I wanna take a screenshot of my Moodle. Do I still have to use my phone to take a picture?
GitHub says that my ID was used before and can't re-use it... Do I have to use other things to verify?

**Reply by devashish2024** [reply]
Yes, you can use something else like a application. You can go through the apply process again and see, where you got to upload ID there below is a dropdown showing what-what ID proofs can be valid. Just try any of them!

> And is your ID latest? You cannot re-use the same ID, ofcourse. Don't try a 2 years old ID, I guess 1 year old can be accepted but 2 is so far

**Reply by conghuan0502** [reply]
well I guess tomorrow I will try again

**Reply by conghuan0502** [reply]
Yea, I tried to upload everything I have, scholarship letter, email, portal screenshot, etc.. but it seems to not work. It said "Please upload proof of your current school affiliation" and now I don't know what to do next.

**Reply by devashish2024** [reply]
Ok, so try to not use your school email this time, maybe?
(it just happens like the approval scores exists...)

**Comment by conghuan0502** [comment]
It's so bad, My school provided me with 2 emails, one for the Vietnam portal and one for the UK portal. I tried almost everything I could think of, but I still can't get into the process stage. What can I do now? I can't write a ticket for manual review either and am seeking help in this forum, but I still can't fix my problem.
![image](https://github.com/community/community/assets/72344330/035a3bb2-6b90-4257-b6e8-1cf7e62d572c)

**Reply by conghuan0502** [reply]
I tried to screenshot my Moodle, upload my scholarship letter, and my financial email and everything keeps being rejected. I still don't know how to fix this problem? So finally I'm not eligible to access the student pack despite I'm learning at school day by day?

**Reply by devashish2024** [reply]
Some schools do have "free accounts" so it gives you a chance but some school students just leave their emails in public, etc it gets to be abused. So, you need a VALID proof of your identity in school, and not a school's photo.

Any ID card from school? Approved application (leave, etc)? Or just ask them for a application for github student dev pack

edit: I mean that some peoples in schools leak their own account for profit and other peoples use it then, because the better school you choose, the better success chances. This all is only for security although.

**Reply by devashish2024** [reply]
Try your personal email

**Reply by conghuan0502** [reply]
After getting rejected 32 times, by using the same Student ID card which got rejected many times. I kept using it and applying till I couldn't. Finally, I can move to the processing stage. Both my edu emails are not public. Proof that I use mostly is scholarship email, ID card, syllabus, etc ... Btw thanks for supporting me.

**Reply by DeveloperrEva** [reply]
❤️

**Comment by DeveloperrEva** [comment]
❤️

**Comment by techdevai-es** [comment]
I cannot register for the student benefits package. My school is listed as "Digital Campus Vocational Training Center", but this school does not provide email accounts to its students. Although the system says that there is no problem and that you can identify yourself with other documents, it does not let me go through the screen if I do not enter a corporate email. What I can do?

**Reply by devashish2024** [reply]
I'm sorry, but all you can do is ask your principal (probably principal because no teachers hold school-level info except principal) and say them to give access or there is nothing you can do.

**Comment by devashish2024** [comment]
> After getting rejected 32 times, by using the same Student ID card which got rejected many times. I kept using it and applying till I couldn't. Finally, I can move to the processing stage. Both my edu emails are not public. Proof that I use mostly is scholarship email, ID card, syllabus, etc ... Btw thanks for supporting me.

No problem but may I ask you, did you continued with personal email or need to get the edu email (I think it continued after 3 days?)

**Reply by conghuan0502** [reply]
Well, I added my edu mail and used it to apply the edu pack.

**Reply by devashish2024** [reply]
Alright!
